src/els_code_action_provider.erl

```diff
@@ -85,10 +85,10 @@ make_unused_variable_action(Uri, Range, UnusedVariable) ->
    } = Range,
   %% processing messages like "variable 'Foo' is unused"
   {ok, #{text := Bin}} = els_utils:lookup_document(Uri),
-  Line = unicode:characters_to_list(els_text:line(Bin, StartLine)),
+  Line = els_utils:to_list(els_text:line(Bin, StartLine)),
```

Comment:
Why would this not be unicode? It should be a line of source code we end up with.

Reply:
The line from the source file could be using some other enconding, right?

In general I replaced every usage of `unicode:characters_to_[list|binary]/1` for this function regardless of the possibility of getting non-unicode in each replacement. It's simpler IMO to have the general rule of not using list_to_binary or binary_to_list and use instead the functions in `els_utils`.

Reply:
ok, just wanted to understand that before diving into this in too much detail. Makes sense.  I guess I should have looked more closely to see that it is calling the new routine now, rather than some standard one.